### ISSUE TITLE
Crash when using mousewheel

### ISSUE THREAD

**Issue by SunShiranui** [HaxePunk/HaxePunk#373]
When using the mousewheel in my game (which **doesn't** use mousewheel controls) I get a crash:

>Invalid field access : released
Called from com/haxepunk/utils/Input.hx line 586
Called from openfl/_legacy/events/EventDispatcher.hx line 98
Called from a C function
Called from openfl/_legacy/display/DisplayObject.hx line 258
Called from openfl/_legacy/display/Stage.hx line 1048
Called from openfl/_legacy/display/Stage.hx line 476

I'm still unsure about the causes of this, or how to reproduce the bug. However, I've tried the following fix, on the onTouchEnd method in Input.hx:

```
private static function onTouchEnd(e:TouchEvent)
	{
		if (_touches.exists(e.touchPointID))
		{
			touches.get(e.touchPointID).released = true;
		}
	}
```
This prevents the crash from happening. I'm not entirely sure whether it breaks anything, so I thought I would submit this as an issue instead of a pull request so someone more familiar with the code could look it over.

**Comment by ibilon** [comment]
Must be the same problem as https://github.com/HaxePunk/HaxePunk/commit/02de41cf97284f701fa77de9c830aa3ad08aa8ec
(we should probably add a fake touch begin event for move if it doesn't exists instead of ignoring it)

I'll add the guard, it's good practice anyway, but we should look into why we miss begin events, and why the mousewheel create a touchEnd event.

**Comment by ibilon** [comment]
The problem is located in lime, I'll close this issue when a new version of it is released.
https://github.com/openfl/lime/commit/6d107b6

**Comment by ibilon** [comment]
It should be fixed in the new lime 2.4.8,
if not please reopen this issue.
